### DISCUSSION TITLE
App doesn't pass flake8 style check on fresh install

### DISCUSSION THREAD

**Discussion by ratchek** [borgbase/vorta#1568] [General]
Hi,
I'm looking to contribute to the vorta codebase (love the app).
I did a fresh install according to the "contributing guide". When I ran the tests with pytest, everything passed. But when I ran the style check with flake8, it came back with lots of errors. Should I just ignore those? And if so, any tips on filtering out only the stuff that I modified so that i don't have to manually look for them?

**Comment by real-yfprojects** [comment]
> But when I ran the style check with flake8, it came back with lots of errors.

Flake passes on my machine and in our CI too.

> And if so, any tips on filtering out only the stuff that I modified so that i don't have to manually look for them?

We use [pre-commit](https://pre-commit.com/).

The contributing guide isn't fully up-to-date any more. Someone has to fix borgbase/vorta.borgbase.com#29.

**Reply by m3nu** [reply]
Right. `make lint` should run the necessary commands. Maybe it didn't use the flake8 configs in the project folder?
